Samples/kotlin/build.gradle

```diff
@@ -44,7 +44,7 @@ dependencies {
     implementation "com.phunware.mapping:mapping:$rootProject.ext.pwMappingSdkVersion"
 
     // Testing
-    androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
+    androidTestImplementation('androidx.test.espresso:espresso-core:3.1.0', {
         exclude group: 'com.android.support', module: 'support-annotations'
```

Comment:
same questions:
do we still need the exclude for com.android.support here?
do we need to change it to android.x?

Reply:
Removed